examples/usage.ipynb

```diff
@@ -260,6 +258,137 @@
    "source": [
     "See the `examples` directory for more examples."
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "# The Apollo architecture\n",
+    "\n",
+    "This section describes the same workflow using the 20-qubit Apollo architecture.\n",
+    "\n",
+    "Qubit connectivity:\n",
+    "```\n",
+    "            QB20   QB17\n",
+    "            /  |   /  |\n",
+    "         QB19  QB16  QB12\n",
+    "        /  |   /  |  /  |\n",
+    "     QB18  QB15  QB11  QB7\n",
+    "        |  /  |  /  |  /\n",
+    "        QB14  QB10  QB6\n",
+    "        /  |  /  |  /\n",
+    "     QB13  QB9    QB5\n",
+    "        |  /  |  /  |\n",
+    "        QB8   QB4   QB2\n",
+    "          |  /  |  /\n",
+    "          QB3   QB1\n",
```

Comment:
I would suggest printing this from the Apollo class docstring.

Reply:
You mean instead of markdown do something like this:

```
apollo = Apollo()
print(apollo.__doc__)
```

?

If so, then probably do the same for Adonis (same notebook, few cells above)?

Reply:
Assuming you meant exactly that: see latest change.